docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -1,466 +1,1741 @@
-.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-.. SPDX-License-Identifier: Apache-2.0
-
-Participant Query Store User Guide
-##################################
+PQS Technical User Documentation
+================================
 
 Introduction
-************
+------------
+
+PQS (Participant Query Store) is a component that extends the reach of
```

Comment:
The docs have this at the beginning:  "The term operational datastore (ODS) usually refers to a database that mirrors the ledger and allows for efficient querying. The Participant Query Store (PQS) feature is an ODS for the participant node."

Can you please add something like this phrasing because customers are very familiar with the term ODS so this will orient them properly (i.e., they will want to keep reading)?